mubench.reviewsite/src/Controllers/RunsController.php

```diff
@@ -488,11 +488,14 @@ private function storeFindings(Detector $detector, Experiment $experiment, $proj
             if(!$misuse){
                 $misuse = $this->createMisuse($detector, $experiment, $projectId, $versionId, $misuseId, $run->id);
             }
-            $this->storeFinding($detector, $experiment, $projectId, $versionId, $misuseId, $misuse, $finding);
+            $last_finding = $this->storeFinding($detector, $experiment, $projectId, $versionId, $misuseId, $misuse, $finding);
+
             if ($experiment->id === 2) {
                 $this->storeFindingTargetSnippets($detector, $projectId, $versionId, $misuseId, $finding['file'], $finding['target_snippets']);
             }
         }
+        $run->updated_at = $last_finding->created_at;
```

Comment:
Doesn't this crash if there are not findings? There should be a test covering this case. Please write one, if there isn't.

Reply:
This method is called only if there are findings.